### DISCUSSION TITLE
warn users when running this program as root?

### DISCUSSION THREAD

**Discussion by sundaran1122** [leo-arch/clifm#14] [Ideas]
by that I don't mean "throw an error" i mean something like

```
$ doas clifm
clifm: You are running clifm as root, this could cause privilage escalation vulnerabilities.

Are you sure you want to continue [Y/n]:
```

**Comment by leo-arch** [comment]
I like it. Sounds fine to me and it's easy to implement. Perhaps we could replace the question by a prompt warning using the `_err` function. Something like this:

    if (root)
        _err('w', PRINT_PROMPT, "%s: You are running as root, this could cause privilege escalation vulnerabilities");

Even more, it won't be a bad idea some kind of more permanent reminder that the root account is being used, changing the color of some element of the interface or something like that.

**Reply by leo-arch** [reply]
Pushed to testing. I added a prompt warning and a permanent bold red 'R' to the beginning of the prompt to remind the user she is running as root. Check it out.
